SocialCareCaseViewerApi.Tests/V1/UseCase/MashReferral/UpdateMashReferralTests.cs

```diff
@@ -62,6 +62,28 @@ public void UpdatingMashReferralThrowsWorkerNotFoundExceptionWhenGetWorkerByWork
                 .WithMessage($"Worker with email \"{request.WorkerEmail}\" not found");
         }
 
+        [Test]
+        public void UpdatingMashReferralUnAssignOnSuccessReturnsMashReferralResponse()
+        {
+            var worker = TestHelpers.CreateWorker();
+            var request = TestHelpers.CreateUpdateMashReferral();
+            request.UpdateType = "UNASSIGN-WORKER";
+
+            var referral = TestHelpers.CreateMashReferral().ToDomain();
+
+            _mashReferralGateway
+                .Setup(x => x.UpdateReferral(request, referral.Id))
+                .Returns(referral);
+
+            var response = _mashReferralUseCase.UpdateMashReferral(request, referral.Id);
+
+            response.Should().BeEquivalentTo(referral.ToResponse(), options =>
+            {
+                options.Using<DateTime>(ctx => ctx.Subject.Should().BeCloseTo(ctx.Expectation, 1000)).WhenTypeIs<DateTime>();
```

Comment:
What do options refer to? Does it affect our response value.

Reply:
In Fluent Assertions when comparing objects with DateTime properties there are sometimes a slight mismatch in the milliseconds and the comparison fail. The way we get around it is to set the comparison option like so.

Reply:
Thanks! 🙌🏾